dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -153,15 +158,27 @@ run_ocp() {
 }
 
 deploy_che_to_ocp() {
-    #Repull init image only if IMAGE_PULL_POLICY is set to Always
-    if [ $IMAGE_PULL_POLICY == "Always" ]; then
-        docker pull "$IMAGE_INIT"
+    #Only generate scripts and config files if CHE_GENERATE_SCRIPTS=true
+    if [ $CHE_GENERATE_SCRIPTS == true ]; then
+      echo "OCP generating temporary scripts and configuration files at ${CONFIG_DIR}/instance/config/openshift/scripts/ ."
+      #Repull init image only if IMAGE_PULL_POLICY is set to Always
+      if [ $IMAGE_PULL_POLICY == "Always" ]; then
+         docker pull "$IMAGE_INIT"
+      fi
```

Comment:
```
 +      if [ $IMAGE_PULL_POLICY == "Always" ]; then
 +         docker pull "$IMAGE_INIT"
 +      fi
```
this must be moved out from `if` statement because it should be considered in both cases

Reply:
Moved.